Clarify guide is for in-place upgrade (#8284)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-upgrade.md

hide_table_of_contents: true
tags: ['dbt Fusion engine', 'dbt platform','Upgrade']
recently_updated: true

> intro_text: This guide is for an in-place upgrade from the latest version of dbt Core to the dbt Fusion engine in the dbt platform.

Reviewer: sugg speaking to the user
```suggestion
intro_text: This guide helps you prepare for an in-place upgrade from the latest version of dbt Core to the dbt Fusion engine in the dbt platform.
```

Author: Great suggestion but going to alter slightly since this guide is for the actual upgrade process

> intro_text: This guide is for an in-place upgrade from the latest version of dbt Core to the dbt Fusion engine in the dbt platform.

Author: ```suggestion
intro_text: This guide helps you execute an in-place upgrade from the latest version of dbt Core to the dbt Fusion engine in the dbt platform.
```

> intro_text: This guide helps you execute an in-place upgrade from the latest version of dbt Core to the dbt Fusion engine in the dbt platform.

Author: ```suggestion
intro_text: This guide helps you implement an in-place upgrade from the latest version of dbt Core to the dbt Fusion engine in the dbt platform.
```
